src/templates/partials/kubelet.hbs

```diff
@@ -0,0 +1,14 @@
+# {{output.header}}
+# {{{output.link}}}
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+tlsCertFile: /path/to/ca.crt
+tlsPrivateKeyFile: /path/to/privkey.pem
+tlsCipherSuites: '{{{join output.ciphers ","}}}'
+{{#if (includes "intermediate" form.config)}}
+tlsMinVersion: TLSv1.2
+{{else if (includes "modern" form.config)}}
+tlsMinVersion: TLSv1.3
+{{else}}
+tlsMinVersion: TLSv1.0
+{{/if}}
```

Comment:
Instead of checking the configuration type like this, check the `output.protocols` variable like you can see here

https://github.com/mozilla/ssl-config-generator/blob/c78c9462a5fbc96dad77ca92defb742092277c9c/src/templates/partials/apache.hbs#L43-L45

Reply:
@atombrella An example with min- value might be e. g.:

https://github.com/mozilla/ssl-config-generator/blob/36671610cd85a132b7c425a9ba2ed0f7057f92fd/src/templates/partials/dovecot.hbs#L19